src/js/hora.js

```diff
@@ -634,7 +646,7 @@ define([
 
                 Hora.__stringifyPropertyValues(transaction);
 
-                Mobify.analytics.ua(ECOMMERCE_PLUGIN + ':addTransaction', transaction);
+                !Hora.isDebug && Mobify.analytics.ua(ECOMMERCE_PLUGIN + ':addTransaction', transaction);
```

Comment:
You're making me wanna proxy Mobify.analytics.ua (like we do above) so we don't need these one-offs, haha.